tests/setup.js

```diff
@@ -0,0 +1,2 @@
+// Global test setup
+jest.setTimeout(10000); // 10 second timeout for tests
```

Comment:
I see we reference this file (`tests/setup.js`) in `jest.config.js` (via `setupFilesAfterEnv` setting).

Here, we are setting the jest test timeout. Why are we doing it like this rather than just setting the timeout in `jest.config.js`? Looking at the [Jest docs](https://jestjs.io/docs/configuration#testtimeout-number), it does look like there is a `testTimeout` confirguration setting we can set for Jest.